### ISSUE TITLE
bug: Can't send ETH

### ISSUE THREAD

**Issue by carletex** [BuidlGuidl/burnerwallet#36]
Wanted to check the latest changes from last week and tried to create a burner, receive some ETH and send some.

Sending ETH doesn't seem to work for me (tried Sepolia & OP Mainnet)
- I see "Sending..." for minutes (need to refresh the page to clear that up and try again)
- Clicking send doesn't seem to fire a TX on the selected network

Is this also happening to you all?

**Comment by ChangoMan** [comment]
Hmm I just tried it, it worked once on Sepolia but then stopped working. Looking through the network logs, I saw an error for insufficient funds but it was a call to `mainnet` instead of `sepolia`.

There might be some funky config thing happening when switching the network, in that the `useSendTransaction` hook still thinks it is on `mainnet` once the saved chainId is loaded from local storage.

I think I can fix it
